qiskit_ibm_runtime/visualization/draw_zne.py

```diff
@@ -0,0 +1,395 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Functions to visualize :class:`~.EstimatorPubResult` ZNE data."""
+
+from __future__ import annotations
+
+from itertools import product
+from typing import Sequence, TYPE_CHECKING
+import numpy as np
+
+from .utils import plotly_module
+from ..utils.estimator_pub_result import EstimatorPubResult
+
+if TYPE_CHECKING:
+    from plotly.graph_objects import Figure as PlotlyFigure
+    from plotly.graph_objects import Scatter as PlotlyScatter
+
+
+def draw_zne(
```

Comment:
Since the other one is called `draw_zne_extrapolators`, I would think of calling this one `draw_zne_evs` or something like that. By reading `draw_zne` I get the impression that this one is "better" than the other one and thus it gets to be "the" draw_zne, but I think they're on equal grounds